python/ccxt/southxchange.py

```diff
@@ -267,6 +280,13 @@ def create_order(self, symbol, type, side, amount, price=None, params={}):
         return {
             'info': response,
             'id': str(response),
+            'symbol': symbol,
+            'type': type,
+            'side': side,
+            'price': price,
+            'amount': amount,
+            'status': 'open',
+            'fee': None,
```

Comment:
fill this field with return value from calculate_fee()

Reply:
fixed